.s2i/bin/run

```diff
@@ -11,16 +11,19 @@
 # SOURCE and HOME DIRECTORY: /opt/app-root/src
 
 #set -x
+INVENTORY="tmp_inventory"
+INVENTORY_ARG="-i ${INVENTORY}"
```

Comment:
Also, the reason I named it `tmp_inventory` was odd.  I need to dig more but when used playbook2image on an inventory in openshift-ansible I soon realized we had a directory called `inventory` that was getting baked in to our image.  I feel like I've seen a few repos with that convention so I renamed it for lack of a better idea.  If we're using playbook2image incorrectly feel free to say so.